src/useComputedStyle/useComputedStyle.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-redeclare */
+import { reactive, ref, Ref, ToRefs, toRefs, watchEffect } from 'vue'
+import { useMutationObserver } from '@/useMutationObserver/useMutationObserver'
+import { useResizeObserver } from '@/useResizeObserver/useResizeObserver'
+import { globalExists } from '@/utilities/global'
+
+export function useComputedStyle(element: Element | Ref<Element | undefined>): ToRefs<Partial<CSSStyleDeclaration>> {
+  const elementRef = ref(element)
+  const style = reactive({})
+
+  watchEffect(() => {
+    if (elementRef.value && globalExists('window')) {
+      Object.assign(style, window.getComputedStyle(elementRef.value, null))
+
+      mutationObserver.disconnect()
+      mutationObserver.observe(elementRef)
+      resizeObserver.disconnect()
+      resizeObserver.observe(elementRef)
+    }
+  })
```

Comment:
I think all of this we only want to happen when `elementRef` changes. I'd convert this to a standard watcher to avoid unnecessary/unintended execution of this callback to do reactivity changes.

Reply:
I think is pretty safe? It only executes if `elementRef.value` is truthy, so first time through when it's probably `undefined`, nothing will happen

Reply:
Right but once's it's truthy it'll run again and then I think it'll start tracking anything reactive inside that if? Could be wrong